environment.sh

```diff
@@ -32,14 +42,6 @@ Help()
    echo "l     Install git lfs."
 }
 
-# Define variables
```

Comment:
just moved this up so I have access to username

Reply:
I like defining the variables sooner, but I don't see where you use username up above